bokeh/resources.py

```diff
@@ -459,5 +454,3 @@ def render(self):
 CDN = Resources(mode="cdn")
 
 INLINE = Resources(mode="inline")
-
-EMPTY = Resources(mode="inline", components=[], log_level=None)
```

Comment:
Pretty sure that at least a few folks have relied on this. Is there a pressing need to remove it?